test/e2e/tests/confirmations/queued-confirmations.spec.ts

```diff
@@ -0,0 +1,423 @@
+/* eslint-disable @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires */
+import { strict as assert } from 'assert';
+import { MockedEndpoint, MockttpServer } from 'mockttp';
+import { getEventPayloads } from '../../helpers';
+import { Driver } from '../../webdriver/driver';
+import { TestSuiteArguments } from './transactions/shared';
+import { MetaMetricsEventName } from '../../../../shared/constants/metametrics';
+
+const FixtureBuilder = require('../../fixture-builder');
+const {
+  withFixtures,
+  openDapp,
+  unlockWallet,
+  DAPP_URL,
+  DAPP_ONE_URL,
+  regularDelayMs,
+  WINDOW_TITLES,
+  defaultGanacheOptions,
+} = require('../../helpers');
+
+describe('Queued Confirmations', function () {
+  if (!process.env.ENABLE_CONFIRMATION_REDESIGN) {
+    return;
+  }
+
+  describe('Existing Requests Banner Alert', function () {
```

Comment:
If we standardise E2E tests for alerts, is it worth a subfolder such as `tests/confirmations/alerts`?